scripts/alert_emails_report/generate-report.ts

```diff
@@ -0,0 +1,96 @@
+import path from 'path';
+import _ from 'lodash';
+import moment from 'moment';
+import CampaignMonitor from '../alert_emails/campaign-monitor';
+import { getFirestore } from '../alert_emails/firestore';
+import { ALERT_EMAIL_GROUP, toISO8601 } from '../alert_emails/utils';
+import fipsByStateCode from '../what-the-fips/2018-census-fips-codes.json';
+import { findStateByFips, getAllStateFips } from '../../src/common/locations';
+import GoogleSheets from '../common/google-sheets';
+
+const REPORT_SPREADSHEET_ID = '1cs3Wyh8Gda0H18_-x5RYp7AJ3FwB-I1ewAQTBWEk1YY';
+
+interface StateFipsMap {
+  [stateCode: string]: {
+    [countyFips: string]: string;
+  };
+}
+
+const keyFile = path.join(
+  __dirname,
+  '../alert_emails/google-service-account.json',
+);
+
+async function generateAlertEmailReport() {
+  const db = getFirestore();
+  const gsheets = new GoogleSheets(keyFile);
+  await gsheets.authenticate();
+
+  // TODO: Confirm, what is the best way to measure engagement stats?
```

Comment:
Like I mentioned, maybe we want unique group IDs?

Reply:
Campaign Monitor says that with ~100 groups it's OK, more than that might not work. Since we only send 1-2 alerts by week, we should have enough for a year or so. I can set a reminder to delete old groups and update the script so we can keep the old stats once we get closer to the limit.